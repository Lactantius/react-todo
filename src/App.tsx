import React from "react";
import "./App.css";
import TodoList from "./TodoList";

function App(): JSX.Element {
  return (
    <div className="App">
      <h1>Todo</h1>
      <TodoList />
    </div>
  );
}

export default App;
