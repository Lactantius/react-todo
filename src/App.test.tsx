import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

jest.mock("uuid", () => ({ v4: () => "123456789" }));

it("renders", () => {
  render(<App />);
});

it("matches snapshot", () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
