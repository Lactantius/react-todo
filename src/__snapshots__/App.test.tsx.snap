// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <h1>
      Todo
    </h1>
    <div
      class="TodoList"
    >
      <form
        class="NewTodoForm"
      >
        <label
          for="text"
        >
          Todo: 
        </label>
        <input
          id="text"
          name="text"
          placeholder="What needs to get done?"
          type="text"
          value=""
        />
        <button>
          New Todo
        </button>
      </form>
      <div
        class="TodoList-container"
      >
        <div
          class="Todo"
          id="123456789"
        >
          <span>
            Make some todos
          </span>
          <button>
            X
          </button>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
